includes/settings/settings-custom-shipping.php

```diff
@@ -15,7 +15,7 @@
 
 $availability_extra_desc_zero  = ' ' . __( 'Ignored if set to zero.', 'custom-shipping-methods-for-woocommerce' );
 $availability_extra_desc_blank = ' ' . __( 'Ignored if empty.', 'custom-shipping-methods-for-woocommerce' );
-$pro_link                      = '<a target="_blank" href="https://wpfactory.com/item/custom-shipping-methods-for-woocommerce/">Custom Shipping Methods for WooCommerce Pro</a>';
+$pro_link                      = '<a target="_blank" href="https://wpfactory.com/item/custom-shipping-methods-for-woocommerce/#distance-shortcode">Custom Shipping Methods for WooCommerce Pro</a>';
```

Comment:
Please change the link here to `https://imaginate-solutions.com/downloads/custom-shipping-methods-for-woocommerce/`